viz/cmd/install.go

```diff
@@ -117,6 +117,12 @@ func install(w io.Writer, options values.Options, ha bool) error {
 		return err
 	}
 
+	// if using -L to specify a non-standard CP namespace, make sure
+	// the linkerdNamespace Helm value is synced
+	if controlPlaneNamespace != "linkerd" {
```

Comment:
Tiny nit: We could probably use the linkerd namespace constant found in `viz/cmd/root.go` on line 17
```suggestion
	if controlPlaneNamespace != defaultLinkerdNamespace
```

Reply:
Good find :+1: